apps/parsley/src/components/LogRow/SkippedLinesRow/SkippedLinesRow.stories.tsx

```diff
@@ -6,53 +6,49 @@ import { LogRenderingTypes, LogTypes } from "constants/enums";
 import { useLogContext } from "context/LogContext";
 import WithToastContext from "test_utils/toast-decorator";
 import { CustomMeta, CustomStoryObj } from "test_utils/types";
-import { ExpandedLine, ExpandedLines } from "types/logs";
-import CollapsedRow from ".";
+import { ExpandedLines, ProcessedLogLines } from "types/logs";
+import SkippedLinesRow from ".";
 
 export default {
-  component: CollapsedRow,
+  component: SkippedLinesRow,
   decorators: [WithToastContext],
-} satisfies CustomMeta<typeof CollapsedRow>;
-
-const CollapsedRowStory = (args: React.ComponentProps<typeof CollapsedRow>) => {
-  const [rows, setRows] = useState(args.collapsedLines);
-  const expandLines = (expandedLines: ExpandedLines) => {
-    const withinRange = (index: number, line: ExpandedLine) =>
-      index >= line[0] && index <= line[1];
-
-    const intervals = rows.filter(
-      (line) =>
-        !withinRange(line, expandedLines[0]) &&
-        !withinRange(line, expandedLines[1]),
-    );
-    setRows(intervals);
+} satisfies CustomMeta<typeof SkippedLinesRow>;
+
+const CollapsedRowStory = (
```

Comment:
Should the story names and components in this file still be making reference to `CollapsedRow`?